packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -170,6 +184,7 @@ const ComboboxField = ({
         ariaLabel={label}
         allowClear={allowClear}
         aria-describedby={ariaDescriptionIds}
+        className={classes}
```

Comment:
If this has an equivalent to inputAttributes, don't forget to combine it with classes.

Reply:
No longer an issue now since `isIncomplete` works more similar to the `isInvalid` prop and the Input control itself adds the class in. https://github.com/cerner/terra-core/pull/2670/commits/93158da0ccc2be1d72588890924879e913215a96